src/components/AbstractModal/AbstractModal.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Modal from 'react-modal';
+
+const AbstractModal = ({ modalId, className, required, modalIsOpen, children }) => {
+    return (
+        <Modal 
+            id={ modalId } 
+            className={ className }
+            shouldCloseOnOverlayClick={ !required }
```

Comment:
I don't think we need to pass required here. I think this value can just be hard coded as false.

Reply:
done